spec/mongo/client_spec.rb

```diff
@@ -584,6 +592,17 @@
         expect(result.first).to eq(filter[:name])
       end
     end
+
+    context 'with comment' do
+      min_server_version '4.4'
+
+      it 'returns a list of database names and send comment' do
+        result = monitored_client.database_names({}, comment: "comment")
+        expect(result).to include('admin')
+        command = subscriber.command_started_events("listDatabases").last&.command
+        expect(command&.fetch("comment")).to eq("comment")
```

Comment:
I would assert that the values here aren't nil, same with other tests.

Reply:
Updated.